src/Bridge/Doctrine/MongoDbOdm/Filter/DateFilter.php

```diff
@@ -107,11 +107,11 @@ protected function filterProperty(string $property, $values, Builder $aggregatio
     /**
      * Adds the match stage according to the chosen null management.
      */
-    private function addMatch(Builder $aggregationBuilder, string $field, string $operator, string $value, string $nullManagement = null): void
+    private function addMatch(Builder $aggregationBuilder, string $field, string $operator, $value, string $nullManagement = null): void
     {
         try {
             $value = new \DateTime($value);
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
This should be reverted. A `\Throwable` should never be thrown. Instead, we should test that the $value is a string before passing it to the `DateTime` constructor.